src/qml/imports/QFieldControls/+Qt6/CalendarPanel.qml

```diff
@@ -27,8 +27,6 @@ Popup {
 
     onAboutToShow: {
```

Comment:
@nirvn what would you suggest doing here, should the `calendar.` lines be commented (so it's easier to revive) or the file minimized to a simple `Item` or similar in order to clearly mark them as "missing implementation". The current state of "80% something" could be misleading?

Reply:
@m-kuhn , TBH, I'd leave it as is, the end result will pretty much match what's in there.

Reply:
I'll trust you on that one